apiserver/facades/agent/provisioner/provisioninginfo_test.go

```diff
@@ -628,67 +627,6 @@ func (s *withoutControllerSuite) TestProvisioningInfoWithUnsuitableSpacesConstra
 	c.Assert(result, jc.DeepEquals, expected)
 }
 
-func (s *withoutControllerSuite) TestProvisioningInfoWithLXDProfile(c *gc.C) {
```

Comment:
Make a note of all the LXD profile tests, as we will want to reinstate them at some point.

Reply:
Sorry this one is actually OK, I won't remove it on this commit, it'll be part of another cleanup.